specs-go/config.go

```diff
@@ -112,12 +112,11 @@ type Platform struct {
 
 // Mount specifies a mount for a container.
 type Mount struct {
-	// Destination is the path where the mount will be placed relative to the container's root.  The path and child directories MUST exist, a runtime MUST NOT create directories automatically to a mount point.
+	// Destination is the absolute path where the mount will be placed in the container.
 	Destination string `json:"destination"`
 	// Type specifies the mount kind.
-	Type string `json:"type,omitempty"`
-	// Source specifies the source path of the mount.  In the case of bind mounts on
-	// Linux based systems this would be the file on the host.
+	Type string `json:"type,omitempty" platform:"linux,solaris"`
```

Comment:
> It confused me because the comment was against `config.go` when really you're referring to `config.md`....

I'm saying that your current `platform:"linux,solaris"` addition here is not as clearly backed by the Markdown spec as I'd like.  I agree that it's where we want the Go type to end up, so I'd like to see your Go change, but I'd *also* like to see the Markdown adjusted to more clearly support the Go change.  In the absence of the Markdown change, I'd rather wait on the Go change.